python/analysis.i

```diff
@@ -0,0 +1,58 @@
+%module analysis
+
+%{
+#define SWIG_FILE_WITH_INIT
+#include <lttoolbox/fst_processor.h>
+#include <lttoolbox/lttoolbox_config.h>
+#include <lttoolbox/my_stdio.h>
+#include <lttoolbox/lt_locale.h>
+
+class FST: private FSTProcessor
+{
+public:
+  /**
+   * Reads from input_path and stores result at output_path
+   */  
+  void init_analysis(char *automorf_path, char *input_path, char *output_path);
+  bool validity() const;
+
+};
+
+bool 
+FST::validity() const
+{
+  return valid();
+}
+
+void 
+FST::init_analysis(char *automorf_path, char *input_path, char *output_path)
```

Comment:
This seems like a misnomer to me since it doesn't just initialize an analysis, it does the whole thing.